dpe/src/x509.rs

```diff
@@ -1030,6 +1160,149 @@ impl X509CertWriter<'_> {
         Ok(bytes_written)
     }
 
+    /// Encode a SignedData
```

Comment:
nit: Might be good to add a comment that we don't populate the `certificates` or `crls` fields

Reply:
Done